internal/provider/cluster_resource.go

```diff
@@ -204,7 +204,7 @@ func (r *clusterResource) Schema(
 					"private_network_visibility": schema.BoolAttribute{
 						Optional:    true,
 						Computed:    true,
-						Description: "Set to true to assign private IP addresses to nodes. Required for CMEK and other advanced networking features.",
+						Description: "Set to true to assign private IP addresses to nodes. Required for CMEK and other advanced networking features. Clusters created with this flag will be Dedicated advanced.",
```

Comment:
@mdlinville this is a very old PR that never landed.  I think its still helpful but the copy is out of date now.  What should it say now?